operator/validator/task_executor.go

```diff
@@ -112,6 +104,5 @@ func (c *controller) UpdateFeeRecipient(owner, recipient common.Address) error {
 		return fmt.Errorf("update validators map: %w", err)
 	}
 
-	logger.Info("executed task")
```

Comment:
when you see the previous log of 'executing task' and no error log after it, it means the task was executed (so 'executed task' log is not necessary)

Reply:
@moshe-blox then maybe better "executed task" and "failed to execute task: ..."? I'd keep "executing task" as debug though to debug a possible deadlock when executing a task